internal/venice-common/src/main/java/com/linkedin/venice/utils/SharedObjectFactory.java

```diff
@@ -12,7 +15,12 @@
  * @param <T> Class whose objects need to release resources cleanly.
  */
 public class SharedObjectFactory<T> {
-  private final Map<String, ReferenceCounted<T>> objectMap = new VeniceConcurrentHashMap<>();
+  private final Map<String, ReferenceCounted<T>> OBJECT_MAP = new VeniceConcurrentHashMap<>();
+  private final Map<String, ReentrantLock> LOCK_MAP = new VeniceConcurrentHashMap<>();
```

Comment:
Nitpick: I think the Java convention is to put in upper case only the `static final` state, not the instance-level state?